safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -7,6 +7,7 @@
 from django.db.models import Min, QuerySet
 
 from celery.exceptions import SoftTimeLimitExceeded
+from requests import ReadTimeout
```

Comment:
Maybe is better to use `Timeout`? https://github.com/psf/requests/blob/cdbc2e271529f467b278b2760f12ee0b5d6930d3/requests/exceptions.py#L61

Reply:
With IOError I'm little bit concerned to include exceptions that cannot be solved reducing the block_process_limit, for me timeout works :)

Reply:
Sounds good